src/mailmojo-settings.php

```diff
@@ -104,29 +123,57 @@ public function create_admin_page () {
 	 * Output section for settings.
 	 */
 	public function settingsSection () {
-		print('Enter the username of your MailMojo Account.');
+		echo '<p>' . sprintf(__('To connect with your MailMojo account, you need to <a href="%s">retrieve an access token</a> from a Wordpress Client in MailMojo. This will enable the widget to retrieve the necessary data to enable signups.', 'mailmojo'),
+			MailMojoSettings::MM_INTEGRATIONS_URL) . '</p>';
+	}
+
+	/**
+	 * Output settings field for access token.
+	 */
+	public function accessTokenField () {
+		$integrationUrl = self::MM_INTEGRATIONS_URL;
+		printf(
+			'<input type="text" id="access_token" name="mailmojo_options[access_token]" ' .
+			'value="%s" placeholder="%s" class="regular-text code"><p><a href="%s">%s</a></p>',
+			isset($this->options['access_token']) ?
+				esc_attr($this->getAccessToken()) : '',
+			__('Paste the access token here', 'mailmojo'),
+			$integrationUrl,
+			__('Get your access token here', 'mailmojo')
+		);
 	}
 
 	/**
 	 * Output settings field for username.
+	 *
+	 * TODO: Remove when username is not needed anymore.
 	 */
 	public function usernameField () {
 		printf(
 			'<input type="text" id="username" name="mailmojo_options[username]" ' .
-			'value="%s">',
+			'value="%s" readonly><p><small><em>%s</em></small></p>',
 			isset($this->options['username']) ?
-				esc_attr($this->options['username']) : ''
+				esc_attr($this->options['username']) : '',
+			__('Deprecated in favor of using the MailMojo API.')
 		);
 	}
 
 	/**
 	 * Sanitize the settings field before saving it.
+	 *
+	 * Saving a field with three stars *** will result in the previous being used.
```

Comment:
Why check for three stars here, but 9 stars is used to obfuscate the access token?